modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -143,6 +144,23 @@ def claims_status_service
         end
       end
 
+      def bgs_service
+        bgs = BGS::Services.new(
+          external_uid: target_veteran.participant_id,
+          external_key: target_veteran.participant_id
+        )
+        ClaimsApi::Logger.log('poa', detail: 'bgs-ext service built')
+        bgs
+      end
```

Comment:
this is bgs-ext - are we safe to get rid of this? or is there still some stuff using it?

Reply:
The v1 validations file [references](https://github.com/department-of-veterans-affairs/vets-api/pull/19620/files#diff-bd4b1830be5b7e6fba3a6dcd3f3c0f9e510f5ec4f746f826780b56395e7c877fR435) this, it is what my PR is working to address

Reply:
There's work in progress under [API-28113](https://jira.devops.va.gov/browse/API-28113) for removing bgs-ext, so I'd rather keep concerns separate from this PR.